### DISCUSSION TITLE
Functions hook in local development

### DISCUSSION THREAD

**Discussion by lukaspili** [supabase/supabase#6696] [General]
It doesn't seem possible at the moment, which makes the development quite painful. 
Is it because of the alpha status?

**Comment by growthondev** [comment]
I am also curious is there any way to do this without the UI?

**Reply by jwright04** [reply]
I used ngrok which allowed me to use function hooks locally (you just have to change the ngrok forwarding address every time you restart the ngrok application).  That's kind of a pain but it works.
